debug-dump/debug-dump.cabal

```diff
@@ -0,0 +1,48 @@
+cabal-version: 1.18
+name: debug-dump
+version: 0.1.0.0
+license: Apache-2.0
+license-file: LICENSE NOTICE
+copyright: Copyright: (c) 2019 Vanessa McHale
+maintainer: vanessa.mchale@iohk.io
+author: Vanessa McHale
```

Comment:
I think we should have IOHK here and in other packages (`language-plutus-core` for example).

Reply:
Yes, you are right.